src/java/org/apache/cassandra/cql3/functions/UDFunction.java

```diff
@@ -266,13 +266,7 @@ public static UDFunction create(FunctionName name,
     {
         assertUdfsEnabled(language);
 
-        switch (language)
-        {
-            case "java":
-                return new JavaBasedUDFunction(name, argNames, argTypes, returnType, calledOnNullInput, body);
-            default:
-                return new ScriptBasedUDFunction(name, argNames, argTypes, returnType, calledOnNullInput, language, body);
```

Comment:
should the `src/java/org/apache/cassandra/cql3/functions/ScriptBasedUDFunction.java` file also be deleted?

Reply:
Totally... forgotten... I will take care, thanks